lib/stat_tracker.rb

```diff
@@ -132,18 +132,43 @@ def worst_offense
     worst_team.team_name
   end
 
-  def highest_scoring_visitor
-    @grouped_game_teams
-    scores_when_visitor = {}
+  def scoring_helper_method(home_or_away_status)
```

Comment:
This helper method is incredible.  Our following methods were much DRYer because of this method.